build/source/engine/var_derive.f90

```diff
@@ -433,16 +433,16 @@ subroutine fracFuture(bpar_data,bvar_data,err,message)
   case(timeDelay)
    ! initialize
    pSave   = 0._rkind ! cumulative probability at the start of the step
-   aLambda = routingGammaShape / routingGammaScale
-   if(routingGammaShape <= 0._rkind .or. aLambda < 0._rkind)then
+   routingGammaMean = routingGammaShape * routingGammaScale
```

Comment:
I agree that this `routingGammaMean` can be removed

Reply:
Hi Martyn, the requested changes have been addressed in my second commit. Please take a look. Thanks.